code/modules/mob/living/simple_animal/borer/egg.dm

```diff
@@ -68,13 +68,23 @@
 /obj/item/weapon/reagent_containers/food/snacks/borer_egg/proc/recruiter_recruited(var/list/args)
 	var/mob/dead/observer/O = args["player"]
 	if(O)
-		var/turf/T = get_turf(src)
-		src.visible_message("<span class='notice'>\The [name] bursts open!</span>")
-		var/mob/living/simple_animal/borer/B = new (T, child_prefix_index)
-		B.transfer_personality(O.client)
-		// Play hatching noise here.
-		playsound(src.loc, 'sound/items/borer_hatch.ogg', 50, 1)
-		qdel(src)
+		var defect = rand(1,10000)
+		if(defect == 666)
```

Comment:
You shouldn't copy paste all this code, and make a variable that gets set to the type of borer to spawn, then use that variable and remove the copy paste.